infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -120,7 +121,7 @@ fi
 git -C $LLVM_SRC checkout $LLVM_REVISION
 echo "Using LLVM revision: $LLVM_REVISION"
 
-# For fuzz introspector.
+For fuzz introspector.
```

Comment:
Unless this is intentionally meant to cause an error, i would guess this should still be a comment.